Fix The Usage Section in Docs (#837)
uclahs-cds/package-moPepGen: docs/split-fasta.md

  --output-prefix path/to/split
```

> This example outputs three split FASTA files, `split_Coding`.fasta`, `split_RNAEditing.fasta`, and `split_Remaining.fasta`.

Reviewer: ?

```suggestion
This example outputs three split FASTA files, `split_Coding.fasta`, `split_RNAEditing.fasta`, and `split_Remaining.fasta`.
```

### Additional split

> Additional split allows you to split the records with the source group specified that would otherwise be placed into the remaining FASTA. See the example below.

Reviewer: ```suggestion
Additional split allows you to split the records with the source group specified that would otherwise be placed along with all other records exceeding `--max-source-groups` into `remaining.fasta`. See the example below.
```

  --output-prefix path/to/split
```

> As a result, `split_gSNP.fasta` and `split_gINDLE.fasta` will be written. `split_gSNP-gINDEL.fasta` and `split_gSNP-RNAEditign.fasta` will also be written although the number of variant sources (2) is larger than the value specified through `--max-source-groups`.

Reviewer: ```suggestion
As a result, `split_gSNP.fasta` and `split_gINDEL.fasta` will be written. `split_gSNP-gINDEL.fasta` and `split_gSNP-RNAEditing.fasta` will also be written although the number of variant sources (2) is larger than the value specified through `--max-source-groups`.
```
